docs/cloud/pouta/ePouta-access.md

```diff
@@ -0,0 +1,24 @@
+# Applying for ePouta access
+
+Applying for **ePouta** access is not a self service process, it might require manual intervention and some **weeks of wait**. This is different from the cPouta, which is self-service and you should be able to get your new project in a mater of minutes. 
```

Comment:
```suggestion
Applying for **ePouta** access is not a self service process, it might require manual intervention and some **weeks of wait**. This is different from the cPouta, which is self-service and you should be able to get your new project in a matter of minutes. 
```